src/o365/graph/commands/planner/task-list.ts

```diff
@@ -0,0 +1,138 @@
+        cmd.log(this.items.map(t => {
+          const task: any = {
+            id: t.id,
+            title: t.title,
+            startDateTime: t.startDateTime,
+            dueDateTime: t.dueDateTime,
+            completedDateTime: t.completedDateTime
+          };
+          return task;
+        }));
+      }
+
+      if (this.verbose) {
+        cmd.log(vorpal.chalk.green('DONE'));
+      }
+      cb();
+    }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--userid [userid]',
+        description: 'The ID of the user to retrieve information for. Specify userid or userName but not both. If none of them are specified, current user tasks will be returned.'
+      },
+      {
+        option: '--userName [userName]',
+        description: 'The username of the user to retrieve information for. Specify userid or userName but not both. If none of them are specified, current user tasks will be returned.'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (args.options.userid && args.options.userName) {
+        return 'Specify either userid or userName but not both. If none of them are specified, current user tasks will be returned.';
+      }
+
+      if (args.options.userid &&
+        !Utils.isValidGuid(args.options.userid)) {
+        return `${ args.options.userid } is not a valid GUID`;
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  ${ chalk.yellow('Important:') } before using this command, log in to the Microsoft Graph
+    using the ${ chalk.blue(commands.LOGIN) } command.
+
+      Remarks:
+
+    To list planner tasks of a user, you have to first log in to
+    the Microsoft Graph using the ${ chalk.blue(commands.LOGIN) } command,
+      eg.${ chalk.grey(`${config.delimiter} ${commands.LOGIN}`) }.
+
+    You can retrieve information about a user, either by specifying that user's
+    userid or user name(${ chalk.grey(`userPrincipalName`) }), but not both.If none of them are specified, current user tasks will be returned.
+
+    If current user don't have permission to retrieve tasks of the specified userid or username, you will get
+    ${ chalk.grey(`You do not have the required permissions to access this item.`) }
+    error.
+
+      Examples:
+
+    List tasks of the current logged in user
+    ${ chalk.grey(config.delimiter) } ${ this.name }
+
+    List tasks of the user with userid ${ chalk.grey(`1caf7dcd-7e83-4c3a-94f7-932a1299c844`) }
+    ${ chalk.grey(config.delimiter) } ${ this.name } --userid 1caf7dcd - 7e83 - 4c3a - 94f7 - 932a1299c844
```

Comment:
It seems like there are some unnecessary spaces in the userId

Reply:
I fixed this, please check.